run_analysis.R

```diff
@@ -32,14 +32,17 @@ source('stan_functions.R')
 # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - #
 # Set parameters
 # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - #
-params = list(rho = c(0.8, 0.9), phi = c(0.5,0.7), sig = c(2.75, 2.2), sig.nu = c(1, 0.8),  sig.0 = c(1.5, 1.3), beta = c(120, 80), cor = 0.55)
-
-attach(params)
-	Sigma.BP = matrix(c(sig[1]^2, cor*sig[1]*sig[2], cor*sig[1]*sig[2], sig[2]^2), nrow = 2)
-	Sigma.nu = matrix(c(sig.nu[1]^2, 0, 0, sig.nu[2]^2), nrow = 2)
-	Sigma.0 = matrix(c(sig.0[1]^2, 0, 0, sig.0[2]^2), nrow = 2)
-	beta = matrix(beta, nrow = 1)
-detach(params)
+params = list(
```

Comment:
Sorry to change this, but I was having some issues with the attach/detach step running properly because it was conflicting with some variables I had loaded in a package I think, so I just removed the attach.